signalfx/resource_signalfx_org_token.go

```diff
@@ -256,7 +277,7 @@ func orgTokenAPIToTF(d *schema.ResourceData, t *orgtoken.Token) error {
 			if limits.DpmNotificationThreshold != nil {
 				dpmStuff["dpm_notification_threshold"] = limits.DpmNotificationThreshold
 			}
-			if err := d.Set("dpm_limit", dpmStuff); err != nil {
+			if err := d.Set("dpm_limits", []map[string]interface{}{dpmStuff}); err != nil {
```

Comment:
What does this change have to do with the auth scope thing?

Reply:
the initial issue reported by the customer is that setting the limit on a token fails with this error:
```
signalfx_org_token.my_test_token: Creating...
╷
│ Error: Invalid address to set: []string{"dpm_limit"}
│
│ with signalfx_org_token.my_test_token,
│ on main.tf line 5, in resource "signalfx_org_token" "my_test_token":
│ 5: resource "signalfx_org_token" "my_test_token" {
```

I was able to reproduce right away and tbh I am not sure how this ever worked, this was broken right from the beginning .
I also noticed that the existing integration test was failing, due to the auth scope.